submissions/Lu-sure/dom-api/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+    padding: 0;
+    margin: 0;
+    font-size: 22px;
+    box-sizing: border-box;
+    font-family: Comfortaa;
+}
+
+html, body {
+    width: 100%;
+    height: 100%;
+}
+
+.wrapper {
+    display: flex;
+}
+
+.side{
+    min-width: 300px;
+    background-color: rgb(37, 49, 116);
+    height: auto;
+}
+
+.side.open {
+    transform: translate(0, 0);
+}
+
+.main {
+    max-height: 1200px;
+}
+
+.side-hero {
+    height: 50px;
+    width: 280;
+    margin: 20px 10px; 
+    padding: 10px;
+    background-color: rgb(201, 187, 221);
+}
+
+.side-hero:hover {
+    background-color: rgb(225, 242, 250);  
+    cursor: pointer;  
+}
+
+.side-button-wrapper {
+    width: 100%;
+    height: 60px;
+    background-image: url(img/bang.jpg);
+    background-position-y: center;
+    display: none;
+}
+
+.side-button {
+    margin: 10px;
+}
+
+.hero {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: rgb(225, 242, 250);
+    height: 100%;
+}
+
+.name {
+    width: 100%;
+    font-size: 2em;
+    font-weight: 700;
+    margin: 20px;
+}
+
+.image {
+    width: 100%;
```

Comment:
Maybe add height/max-height for image (or add wrapper for image) ?
After your render function insert html into your main block, image fetched src and started to download. This state looks like in this image. 
![image](https://user-images.githubusercontent.com/33497965/99876411-36254880-2bff-11eb-9357-171450b543e3.jpeg)

When download completed it looks like this one.
![image2](https://user-images.githubusercontent.com/33497965/99876436-65d45080-2bff-11eb-9bee-6cd69c9b6e41.jpeg)

So it causes expanding of your block. And visually looks like "blink".

Reply:
Fixed.